packages/salesforcedx-vscode-core/src/conflict/conflictUtils.ts

```diff
@@ -34,22 +36,6 @@ export const filesDiffer = async (one: string, two: string): Promise<boolean> =>
   return buffer1 !== buffer2;
 };
 
-/**
```

Comment:
this is why we `npx knip`.

`export` hides dead code.  AI will generate a lot of it and just doesn't care.